### DISCUSSION TITLE
vn7.6 released

### DISCUSSION THREAD

**Discussion by ericaneininger** [MetOffice/simulation-systems#214] [JULES]
I am pleased to announce that JULES vn7.6 has been released.

Thank you to all developers, code owners and reviewers for their contributions.

Documentation and release notes are available at http://jules-lsm.github.io/

Thank you,
Erica, on behalf of the SSD Team.

**Comment by jennyhickson** [comment]
Apologies - a small bug had been found in the release. This has now been corrected and the tagged revision moved. If you have already branched from vn7.6 then you may need to redo this. 

Sorry for any problems or confusion. 
Jenny
